File: php/scripts/location_types.php
<?php
require_once(__DIR__."/../vendor/autoload.php");
require_once("utils.php");
use Namelivia\TravelPerk\ServiceProvider;
$isSandbox = false;
$travelperk = (new ServiceProvider())->build(getenv("API_KEY"), false);
$locationTypes = $travelperk->travelSafe()->travelSafe()->locationTypes();
write_output("location_types", [
	$locationTypes
]);

File: python/scripts/location_types.py
import os
from travelperk_http_python.builder.builder import build
from utils import write_output
travelperk = build(os.environ["API_KEY"], False)
location_types = travelperk.travelsafe().travelsafe().location_types()
write_output("location_types", [
    location_types
])
